CHANGELOG.md

```diff
@@ -9,36 +9,46 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - ReleaseDate
 
+- Add `get_bank_by_bic` to fetch a bank by its BIC
+
 ## [1.0.6] - 2020-11-09
+
```

Comment:
Please remove these spaces, it throws off my changelog tool. :P

Reply:
Funny, I've used a code formatter to apply common markdown formatting. Might be worth to fix that :)

Reply:
but sure.. can do it